model_servers/object_detection_python/src/object_detection_server.py

```diff
@@ -9,7 +9,9 @@
 
 
 app = FastAPI()
-model = os.getenv("MODEL_PATH", default="facebook/detr-resnet-101")
+models_path = os.getenv("MODELS_PATH", default="/app/models")
+model_name = os.getenv("MODEL_NAME", default="detr-resnet-101")
+model = os.path.abspath(f"{models_path}/{model_name}")
```

Comment:
I don't think that this is compatible with the way AI Lab will expect the model paths to be. Can you revert to the original. Jeff is changing this file in #256 as well.